tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -184,34 +185,19 @@ def get_logs(self, start, end):
     def validate(self):
         """Validate configuration settings for execution mode."""
 
-    def _build_functions_package(self, queue_name):
-        package = FunctionPackage(self.policy_name,
-                                  )
+    def build_functions_package(self, queue_name=None):
+        self.log.info("Building function package for %s" % self.function_params.function_app_name)
+
+        package = FunctionPackage(self.policy_name)
         package.build(self.policy.data,
-                      modules=['c7n', 'c7n-azure'],
+                      modules=['c7n', 'c7n-azure', 'applicationinsights'],
```

Comment:
You can move applicationinsights to the setup.py (currently it is installed as azure-cli-core dependency)